# Set the extrinsics info for this manager

ExtrinsicsInfo info
---
